File: src/main/Java/boj_step5/boj2675.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2675 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int t = Integer.parseInt(reader.readLine());
        while (t-- > 0) {
            StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
            int r = Integer.parseInt(tokenizer.nextToken());
            char[] s = tokenizer.nextToken().toCharArray();
            for (char c: s) {
                int index = r;
                while (index-- > 0) writer.append(c);
            }
            writer.newLine();
        }
        writer.close();
    }
}

File: src/main/kotlin/boj2675.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val T = r.readLine()!!.toInt()
    repeat(T) {
        val (n, s) = r.readLine()!!.split(" ").map { it }
        val r = n.toInt()
        for (i in 0 until s.length) w.write(s.get(i).toString().repeat(r))
        w.write("\n")
    }
    w.close()
}